Library/Homebrew/extend/fileutils.rb

```diff
@@ -6,24 +7,15 @@ module FileUtils
   # Create a temporary directory then yield. When the block returns,
   # recursively delete the temporary directory.
   def mktemp(prefix=name)
-    # I used /tmp rather than `mktemp -td` because that generates a directory
-    # name with exotic characters like + in it, and these break badly written
-    # scripts that don't escape strings before trying to regexp them :(
+    Dir.mktmpdir(prefix, HOMEBREW_TEMP) do |tmp|
```

Comment:
This obv. doesn't need to hold this up, but arg, ruby 1.8.2 doesn't have `Dir.mktmpdir`. :crying_cat_face: